app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryOwnedBy.js

```diff
@@ -1,24 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import SelectUser from '../select_user/SelectUser';
-import { editingStoryPropTypesShape } from '../../../models/beta/story';
+import { canEdit, editingStoryPropTypesShape } from '../../../models/beta/story';
```

Comment:
Looks like we're not using `canEdit` here?

Reply:
True, it was refactored and I forgot about that